ktor-server/ktor-server-host-common/jvm/src/io/ktor/server/engine/BaseApplicationEngine.kt

```diff
@@ -31,20 +31,34 @@ public abstract class BaseApplicationEngine(
     public open class Configuration : ApplicationEngine.Configuration()
 
     init {
+        var isFirstLoading = true
+        var initializedStartAt = System.currentTimeMillis()
         BaseApplicationResponse.setupSendPipeline(pipeline.sendPipeline)
         environment.monitor.subscribe(ApplicationStarting) {
+            if (!isFirstLoading) {
+                initializedStartAt = System.currentTimeMillis()
+            }
             it.receivePipeline.merge(pipeline.receivePipeline)
             it.sendPipeline.merge(pipeline.sendPipeline)
             it.receivePipeline.installDefaultTransformations()
             it.sendPipeline.installDefaultTransformations()
             it.installDefaultInterceptors()
         }
         environment.monitor.subscribe(ApplicationStarted) {
+            val finishedAt = System.currentTimeMillis()
             environment.connectors.forEach {
                 environment.log.info(
                     "Responding at ${it.type.name.lowercase(Locale.getDefault())}://${it.host}:${it.port}"
                 )
             }
+
+            val elapsedTimeInSeconds = (finishedAt - initializedStartAt) / 1_000.0
+            if (isFirstLoading) {
+                environment.log.info("Application started in $elapsedTimeInSeconds seconds.")
+                isFirstLoading = false
+            } else {
+                environment.log.info("Auto-reload application in $elapsedTimeInSeconds seconds.")
```

Comment:
Sorry, I forgot to add a comment in the last review. Same as in the previous log message, `Application auto-reloaded`

Reply:
Exactly, I should have guessed about it myself; /)